### PR TITLE
Transforming the canvas broke when there was more than one layer and …

### PR DESCRIPTION
**Pull request by andybak**
…the activate canvas wasn't the default one.

### CODE REVIEW

**Comment by andybak** [comment]
Layers were partially implemented back in google days - but very unfinished with no UI
I think they might have intended to use two hand grab to move canvases and some of that logic was still in there.

The first change in this file removes that logic.

The second one just applies any canvas transforms to all layers.

**Review by cdadashovkhandan — APPROVED** [review]
Works perfectly for me now.

**Comment by andybak** [comment]
I wonder if we should remove the InGrabCanvasMode property entirely...

File: Assets/Scripts/SketchControlsScript.cs [hunk 1]
```diff
@@ -724,15 +724,6 @@ public bool InGrabCanvasMode
...
-#if (UNITY_EDITOR || EXPERIMENTAL_ENABLED)
-                if (Config.IsExperimental)
-                {
-                    if (App.Scene.ActiveCanvas != App.Scene.MainCanvas)
-                    {
-                        return true;
-                    }
-                }
-#endif
                 return false;
```
Comment (merulaxr) [inline Assets/Scripts/SketchControlsScript.cs:727]: With this change, isn't `InGrabCanvasMode` always false? If so, the added code below never runs.

Comment (andybak) [reply Assets/Scripts/SketchControlsScript.cs:727]: Yeah. I was thinking the same myself. At the time I was erring on the side of "least changes" but InGrabCanvasMode and the conditional that uses it do nothing now.

Comment (merulaxr) [reply Assets/Scripts/SketchControlsScript.cs:727]: if it's not used anywhere else, I'd suggest that we just get rid of it, unless you could see it being used elsewhere later? Though, if it breaks things, perhaps it's best to remove now and come back to the idea entirely when we *need* to :D

**Review by merulaxr — APPROVED** [review]
Thanks! @andybak will you be adding the Layers opening button in this PR or a separate one?

**Pull request merged**